### DISCUSSION TITLE
Calendar view default to record list

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#14218] [How to] [answered]
Hello,

Is there any option where I can set Calendar View default to List View instead of Graphical Dashboard view.

![image](https://user-images.githubusercontent.com/12723595/107327861-d5a5f900-6ad3-11eb-848d-d6ed2598556e.png)

Regards
Jeevan Pendhari

**Comment by rskrzypczak** [comment, marked as answer]
Yes, there is. The first and easiest option is to just set a shortcut to the list in the menu configuration.
![image](https://user-images.githubusercontent.com/10221999/107331201-d4b09f80-6ab2-11eb-8cc6-62f5e69b39a8.png)

The second option is unfortunately to change the default view of the module in the file.
https://github.com/YetiForceCompany/YetiForceCRM/blob/6e19188d06ee7a525b8ae33616ef45b3d82bdc5d/modules/Calendar/models/Module.php#L23-L26

**Comment by JEEVAN28** [comment]
Greate !!! First option worked for me 

Thanks
